src/components/Help/TooltipAndRTDLink/index.tsx

```diff
@@ -1,31 +1,35 @@
 import React from 'react';
-import T from 'prop-types';
 
 import config from 'config';
 import Link from 'components/generic/Link';
+
 // $FlowFixMe
 import Tooltip from 'components/SimpleCommonComponents/Tooltip';
 
-import { foundationPartial } from 'styles/foundation';
-import interproTheme from 'styles/theme-interpro.css'; /* needed for custom button color*/
 import fonts from 'EBI-Icon-fonts/fonts.css';
-import ipro from 'styles/interpro-new.css';
+import local from './style.css';
+
+import cssBinder from 'styles/cssBinder';
+const f = cssBinder(local, fonts);
```

Comment:
It's not super important, but traditionally the `f` constant (which comes from Foundation, the old CSS framework) is renamed to `css`.

Reply:
I also do it because I don't know what extension. highlights the usage of `f` as a potential using a system function.